conference/management/commands/create_initial_data_for_dev.py

```diff
@@ -222,3 +245,200 @@ def new_page(rev_id, title, **kwargs):
             status=News.STATUS.DRAFT,
             published_date=None,
         )
+
+        create_tags()
+
+
+def create_tags():
+    """
+    Uses dump for pre-2019 data to populate ConferenceTag instances for CFP
+    """
+
+    INPUT = """
```

Comment:
Keeping the data in a separate file would be cleaner, but it's a minor thing (on top of this being a dev tool) :)